client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -2,18 +2,22 @@
 # Set up DCR and BTC wallets and register with the DEX.
 # dcrdex, dexc, and the wallet simnet harnesses should all be running before
 # calling this script.
+
+RPC_PASS="abc"
+
 set -e
+AUTH="./dexcctl -p abc -P ${RPC_PASS}"
 echo initializing
-./dexcctl -p abc init
+${AUTH} init
 echo configuring Decred wallet
-./dexcctl -p abc -p abc newwallet 42 ~/dextest/dcr/alpha/alpha.conf '{"account":"default"}'
+${AUTH} -p abc newwallet 42 ~/dextest/dcr/alpha/alpha.conf '{"account":"default"}'
 echo configuring Bitcoin wallet
-./dexcctl -p abc -p "" newwallet 0 ~/dextest/btc/alpha/alpha.conf '{"walletname":"gamma"}'
+${AUTH} -p "" newwallet 0 ~/dextest/btc/alpha/alpha.conf '{"walletname":"gamma"}'
 echo configuring Litecoin wallet
-./dexcctl -p abc -p "" newwallet 2 ~/dextest/ltc/alpha/alpha.conf '{"walletname":"gamma"}'
+${AUTH} -p "" newwallet 2 ~/dextest/ltc/alpha/alpha.conf '{"walletname":"gamma"}'
 echo registering with DEX
-./dexcctl -p abc register 127.0.0.1:17273 100000000 ~/dextest/dcrdex/rpc.cert
+${AUTH} register 127.0.0.1:17273 100000000 ~/dextest/dcrdex/rpc.cert
```

Comment:
I had forgotten the brackets here, but it looks like my bash didn't need them...